packages/components/src/components/Map.tsx

```diff
@@ -14,6 +14,11 @@ import 'leaflet/dist/leaflet.css';
 import * as L from 'leaflet';
 
 export type MapProps = {
+  tile?: {
+    attribution?: string;
+    url: string;
+    data?: any;
```

Comment:
Is this prop used anywhere?

Reply:
now this is all defined in TileLayerPresets object